docs/source/reference/numpysupported.rst

```diff
@@ -53,9 +53,38 @@ The following scalar types and features are not supported:
 * **Half-precision and extended-precision** real and complex numbers
 * **Nested structured scalars** the fields of structured scalars may not contain other structured scalars
 
-The operations supported on scalar Numpy numbers are the same as on the
-equivalent built-in types such as ``int`` or ``float``.  You can use
-a type's constructor to convert from a different type or width.
+The operations supported on scalar Numpy numbers are almost the same as on the
+equivalent built-in types such as ``int`` or ``float``.  You can use a type's
+constructor to convert from a different type or width. In addition you can use
+the ``view(np.<dtype>)`` method to bitcast all ``int`` and ``float`` types
+within the same width. However, you must define the scalar using a Numpy
+constructor within a jitted function. So for example, the following will work:
+
+.. code:: pycon
+
+    >>> import numpy as np
+    >>> from numba import njit
+    >>> @njit
+    ... def bitcast():
+    ...     i = np.int64(-1)
+    ...     print(i.view(np.uint64))
+    ...
+    >>> bitcast()
+    18446744073709551615
+
+
+Whereas the following will not work:
+
+
+.. code:: pycon
+
+    >>> import numpy as np
+    >>> from numba import njit
+    >>> @njit
+    ... def bitcast(i):
+    ...     print(i.view(np.uint64))
+    ...
+    >>> bitcast(np.int64(-1))
```

Comment:
Probably best to either remove this line, or show the exception string, else it adds nothing?

Reply:
yes